tests/system-tests/ipsec/README.md

```diff
@@ -0,0 +1,228 @@
+
+1. The `iperf3` binary is not available on the Security Gateway
+2. A firewall on the Security Gateway is blocking the `iperf3` ports. Refer to Appendix A4.
+3. The iperf3 image for the SNO cluster is not available. Refer to Appendix A3.
+4. One of the IPs in `eco-gotests/tests/system-tests/ipsec/internal/ipsecconfig/default.yaml`
+   is incorrect.
+
+## Appendix
+
+### A1 Checking the IPSec tunnel
+
+The following 3 commands show different information about the IPSec tunnel:
+
+```
+sudo ipsec briefstatus
+sudo ipsec showstates
+sudo ipsec traffic
+```
+
+The following command shows much more information, and usually will not be necessary
+unless more involved debugging is needed:
+
+```
+sudo ipsec briefstatus
+```
+
+It must be possible to ping the Security Gateway VPN IP address from the SNO cluster.
+Notice this ping must be performed with the correct source IP, as follows:
+
+```
+# On the Security Gateway, get the VPN IP Address:
+ip a show ipsec
+5: ipsec: <NO-CARRIER,POINTOPOINT,MULTICAST,NOARP,UP> mtu 1500 qdisc fq_codel state DOWN group default qlen 500
+    link/none 
+    inet 172.16.123.10/24 brd 172.16.123.255 scope global noprefixroute ipsec
+       valid_lft forever preferred_lft forever
+```
+
+```
+# On the SNO cluster, get the source IP address:
+ip a show br-ex noprefixroute
+14: br-ex: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/ether 6c:fe:54:58:ee:e4 brd ff:ff:ff:ff:ff:ff
+    inet 10.1.101.129/27 brd 10.1.101.159 scope global noprefixroute br-ex
+       valid_lft forever preferred_lft forever
+
+# On the SNO cluster Ping the Security Gateway, via the IPSec tunnel
+ping -I 10.1.101.129 172.16.123.10
+PING 172.16.123.10 (172.16.123.10) from 10.1.101.129 : 56(84) bytes of data.
+64 bytes from 172.16.123.10: icmp_seq=1 ttl=64 time=0.635 ms
+64 bytes from 172.16.123.10: icmp_seq=2 ttl=64 time=0.381 ms
+64 bytes from 172.16.123.10: icmp_seq=3 ttl=64 time=0.277 ms
+```
+
+### A2 Creating the OCP IPSec Machine Config
+
+Complete instructions can be found here:
+
+```
+cnf-features-deploy/ztp/source-crs/optional-extra-manifest/README.md
+```
+
+It is best to use butane version `0.20.0`.
+
+The most important parts of this is to place the latest certificates in this
+directory, and to correctly configure the IP Addresses in the `ipsec-endpoint-config.yml`
+file.
+
+Once the Machine Configurations have been generated, store the updated `99-ipsec-master-endpoint-config.yaml`
+file in the SNO cluster ZTP git repo.
+
+### A3 The iperf3 ubi image
+
+Since the `iperf3` application is not available on RHCOS, an image was created so
+the application can be used for these test cases. This image was created with the
+following Dockerfile: `eco-gotests/images/system-tests/ipsec-iperf3/Dockerfile`.
+
+The image should be stored as can be seen by this environment variable:
+
+```
+export ECO_IPSEC_TESTS_IPERF3_IMAGE=registry.kni-qe-62.lab.eng.rdu2.redhat.com:5000/iperf3:ubi9
```

Comment:
Could you please update `kni-qe-62.lab.eng.rdu2.redhat.com` with a generic domain name such as  `example.com`?